src/cephadm/cephadm

```diff
@@ -4019,6 +4020,10 @@ def _get_parser():
     parser_install = subparsers.add_parser(
         'install', help='install ceph package(s)')
     parser_install.set_defaults(func=command_install)
+    parser_install.add_argument(
+        'packages', nargs='*',
+        default=['ceph-common', 'cephadm'],
```

Comment:
what about having an `alias ceph="ceph shell --"` instead of installing ceph-common?

Reply:
Hmm... yeah, that would work for the install procedure at least.

Maybe for this PR we make the install command default to the 'cephadm' package only?